src/moments.jl

```diff
@@ -63,10 +63,18 @@ function var!(R::AbstractArray, A::RealArray, wv::WeightVec, dim::Int; mean=noth
 end
 
 Base.varm(A::RealArray, wv::WeightVec, M::RealArray, dim::Int) =
-    Base.varm!(Array(Float64, Base.reduced_dims(A, dim)), A, wv, M, dim)
+    @static if VERSION < v"0.5.1-pre+19"
+        Base.varm!(similar(A, Float64, Base.reduced_dims(size(A), dim)), A, wv, M, dim)
+    else
+        Base.varm!(similar(A, Float64, Base.reduced_indices(indices(A), dim)), A, wv, M, dim)
```

Comment:
ignore what's on the tip of release-0.5. if there's a breakage it won't be included in 0.5.1 release, it's a base bug.